src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/EventLoggingService.kt

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.hmpps.personrecord.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.personrecord.jpa.entity.EventLoggingEntity
+import uk.gov.justice.digital.hmpps.personrecord.jpa.repository.EventLoggingRepository
+import java.time.LocalDateTime
+
+@Service
+class EventLoggingService(
+  private val eventLoggingRepository: EventLoggingRepository,
+
+) {
+
+  fun mapToEventLogging(
+    operationId: String? = null,
+    beforeData: String? = null,
+    processedData: String? = null,
+    sourceSystemId: String? = null,
+    uuid: String? = null,
+    sourceSystem: String? = null,
+    messageEventType: String? = null,
+    eventTimeStamp: LocalDateTime? = null,
+  ): EventLoggingEntity {
+    val eventLog = EventLoggingEntity(
+      beforeData = beforeData,
+      processedData = processedData,
+      sourceSystemId = sourceSystemId,
+      uuid = uuid,
+      sourceSystem = sourceSystem,
```

Comment:
sure I have asked this before and forgotten the answer, but why do we have a sourceSystem and a sourceSystemId?